modules/http/src/main/scala/higherkindness/mu/http/implicits.scala

```diff
@@ -65,7 +65,10 @@ object implicits {
         decoder: Decoder[A],
         F: ApplicativeError[F, Throwable]
     ): Stream[F, A] =
-      message.body.chunks.parseJsonStream.map(_.as[A]).rethrow
+      message.body.chunks.parseJsonStream
+        .flatMap(
+          _.as[List[A]].map(Stream.emits(_)).fold(_.raiseError[Stream[F, *], A], identity(_))
```

Comment:
What if we use this? It's almost the same, but single elements are emitted by `jawn-fs2` instead of handling that ourselves. I didn't find exactly why chunks are now received this way, there was a change in how chunks are backed (predef `Seq` between Scala 2.12 and 2.13 changed from mutable to immutable) but nothing specific to this...

```suggestion
      message.body.chunks.unwrapJsonArray.map(_.as[A]).rethrow
```

Tests seem to be passing.

Reply:
🤔  interesting. My concern is why we're receiving the array instead of separated elements. That could potentially affect to the performance unless I'm missing something, like data be grouped in chunks due to how they are generated. Anyway, your change makes sense to me.